### ISSUE TITLE
Loss setting: can we mix likelihood and RMSE together?

### ISSUE THREAD

**Issue by waystogetthere** [SimiaoZuo/Transformer-Hawkes-Process#15]
In the Paper the Loss function contains 3 parts:
<img width="551" alt="image" src="https://user-images.githubusercontent.com/30395015/202897371-12ba0069-e65a-4999-aae8-129c364b178f.png">

The first part is the log-likelihood, the second is the event cross-entropy loss, and the third is the time RMSE loss.

I don't know if it is appropriate to mix these all together. As I expect, if the log-likelihood is applied, then to predict the next event's time, the only way is to calculate the expectation of PDF:
<img width="309" alt="image" src="https://user-images.githubusercontent.com/30395015/202897254-af54e1d1-5d1b-4346-b3bb-1de7b74b5739.png">

Can anyone help me make further clarification?
